evap/evaluation/migrations/0010_fill_textanswer_state.py

```diff
@@ -28,6 +30,7 @@ def backward_update_states(apps, schema_editor):
             answer.checked = False
         answer.save()
 
+
 class Migration(migrations.Migration):
 
```

Comment:
could you also make these empty lines at the beginning of a class consistent? i'd prefer having none there.

Reply:
i don't see us enforcing that during review, or would you do that? and i cannot find a lint that would enforce this. then i would rather not care about this altogether...

Reply:
i just thought since you're at it. sure, we definitely can't enforce that during review when there is no lint checking it because we will overlook it eventually. i mean in most cases i would notice it if someone adds it because it annoys me, but that's not guaranteed. okay then, look like we keep it.